HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -810,4 +812,25 @@ public PathByActionArguments(string action, string controller, RouteValueDiction
             Values = values;
         }
     }
+    
+    private static string GetLocalAuthorityPendingMessagePartialViewPath(
+        Questionnaire questionnaire)

```

Comment:
No need to break a line for the parameter, now that it's short enough. Same below.